### DISCUSSION TITLE
[QUESTION] SSL: UNEXPECTED_MESSAGE

### DISCUSSION THREAD

**Discussion by rvaidun** [aio-libs/aiokafka#1083] [Q&A]
Hi All,
My consumers that are running for a long time are not receiving messages that are sent to Kafka. I am seeing lots of log such as this in my consumer services:
```
Unable connect to node with id 0: [SSL: UNEXPECTED_MESSAGE] unexpected message (_ssl.c:1149)
Unable connect to node with id 1: [SSL: UNEXPECTED_MESSAGE] unexpected message (_ssl.c:1149)
Unable connect to node with id 2: [SSL: UNEXPECTED_MESSAGE] unexpected message (_ssl.c:1149)
```
https://github.com/aio-libs/aiokafka/blob/256ce17c2d235f417b5302a6046c27879ce610f7/aiokafka/client.py#L475-L481
Would this be causing issues with receiving messages?

**Comment by amabilee** [comment]
Ensure that the certificates and keys are correctly configured for both Kafka brokers and consumers.
Make sure you are using the latest version of the aiokafka.
Any potential network issues or firewalls that might be interfering with the SSL communication.
